delta/testkit/src/main/scala/ch/epfl/bluebrain/nexus/testkit/mu/ce/CatsEffectAssertions.scala

```diff
@@ -230,7 +230,7 @@ trait CatsEffectAssertions { self: Assertions =>
       *   IO.unit.assert // OK
       * }}}
       */
-    def assert(implicit loc: Location): IO[Unit] =
+    def assertUnit(implicit loc: Location): IO[Unit] =
```

Comment:
This is coming from the Cats Effect Munit library.
When we use CE3, it is likely that we use directly the library and get rid of this class.
Could we keep the same name for now ?

Reply:
Can I merge this and rename again in another PR? I've got a backlog